aten/src/ATen/core/interned_strings.h

```diff
@@ -279,6 +280,9 @@ namespace c10 {
   _(aten, zero_)                     \
   _(aten, fill_)                     \
   _(aten, masked_fill_)              \
+  _(cuda, _cuda_setDevice)           \
+  _(cuda, _cuda_setStream)           \
+  _(aten, current_device)            \
```

Comment:
How come this is not in the `cuda` namespace? I think you tried but I'm curious about the explanation.

Reply:
So for current_device, using the namespace `cuda` , conflicts with the `c10::cuda::current_device`. Hence, I couldn't add it in the `cuda` namespace. If we want to change `_cuda_setDevice` -> `set_device` or all these api's to be similar to api's in `c10::cuda::` namespace , then we cannot use the cuda namespace. We then have to consider changing the name of the API or change namespace